Formula/awscli@1.rb

```diff
@@ -4,8 +4,8 @@ class AwscliAT1 < Formula
   desc "Official Amazon AWS command-line interface"
   homepage "https://aws.amazon.com/cli/"
   # awscli should only be updated every 10 releases on multiples of 10
-  url "https://github.com/aws/aws-cli/archive/1.20.10.tar.gz"
-  sha256 "25cc31a926f43761cf17689ca4558e2f73d956a02064cabc265c6dd10aeb15f9"
+  url "https://github.com/aws/aws-cli/archive/1.20.20.tar.gz"
```

Comment:
Do we want to try using PyPI and adding resources? https://files.pythonhosted.org/packages/b0/ad/4916bfa9789428c5c801467f8281d8eb82fda620ba0e7c2dee803df94195/awscli-1.20.20.tar.gz

Unlike #83109, `awscli@1` is available on PyPI. It also doesn't use `awscrt`, so we shouldn't hit issue in other PR.